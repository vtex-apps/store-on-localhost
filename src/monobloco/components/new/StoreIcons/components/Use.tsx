import React from 'react'

interface Props {
  id: string
}

const Use = ({ id }: Props) => <use href={`#${id}`} xlinkHref={`#${id}`} />

export default Use
