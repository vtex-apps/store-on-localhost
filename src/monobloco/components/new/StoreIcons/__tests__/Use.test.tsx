import React from 'react'
import { render } from '@vtex/test-tools/react'
import Use from '../components/Use'

describe('Use', () => {
  it('should match snapshot', () => {
    const component = render(
      <svg>
        <Use id="tst-snapshot" />
      </svg>
    ).asFragment()
    expect(component).toMatchSnapshot()
  })
})
