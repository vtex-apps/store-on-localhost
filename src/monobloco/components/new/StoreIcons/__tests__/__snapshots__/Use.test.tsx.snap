// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Use should match snapshot 1`] = `
<DocumentFragment>
  <svg>
    <use
      href="#tst-snapshot"
      xlink:href="#tst-snapshot"
    />
  </svg>
</DocumentFragment>
`;
